engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -335,7 +335,7 @@ class EngineJobExecutionActor(replyTo: ActorRef,
   }
 
   private def writeToMetadata(keyValues: Map[String, String]) = {
-    import cromwell.services.metadata.MetadataService.implicits.MetadataAutoputter
+    import cromwell.services.metadata.MetadataService.implicits.MetadataAutoPutter
```

Comment:
Hard to unsee: https://www.amazon.com/Pinemeadow-Golf-Sport-Auto-Putter/dp/B008OTMGYC